drivers/iio/light/ltr.c

```diff
@@ -1,8 +1,9 @@
 // SPDX-License-Identifier: GPL-2.0-only
 /*
- * ltr501.c - Support for Lite-On LTR501 ambient light and proximity sensor
+ * ltr.c - Support for Lite-On LTR501, LTR509, LTR301, LTR303 ambient light and proximity sensors (only LTR5xx)
```

Comment:
As mentioned on the previous commit, this is a standard Linux file that hasn't been altered in our branches - this change must be sent to the main kernel maintainers.